### PR TITLE
Monkey Powder Part.2

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
Changes monkey powder to be more realistic when reacting inside mobs. Also makes it take more monkey powder to make a monkey as to not have a redo of itching powder
-Gibs monkeys when it reacts inside of them
-Makes regular carbons vomit out a monkey

## Why It's Good For The Game
This was supposed to be rolled into the original monkey powder pr but didn't make it in to not ruffle any feathers.

## Changelog
:cl:
tweak: Changes monkeypowder's behavior in carbons
/:cl:

### CODE REVIEW

**Comment by Winterous** [comment]
I love exploding monkes with other monkes. But what do you mean by a re-do of itching powder?

**Comment by spessbro** [comment]
> what do you mean by a re-do of itching powder?

Itching powder was a meme chem like this but became a meta chem because it did tox and brute and was generally annoying

**Comment by IndieanaJones** [comment]
This seems like it'll wreck player monkeys pretty hard, which kinda sucks.

**Comment by Barhandar** [comment]
Players who embrace monke deserve to suffer.

**Comment by spessbro** [comment]
> This seems like it'll wreck player monkeys pretty hard, which kinda sucks.

I was planning on actually checking for a ckey but the thing is that it takes a minimum of 51 units of monkey powder and water to actually have the reaction work and if you look at stuff like mint toxin which does need a specific condition but also straight up gibs with only 1 unit

**Comment by Winterous** [comment]
As a solution to gib-syringing monkies, maybe you could make it only work inside the stomach, not the bloodstream.

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -431,19 +431,23 @@
...
-	var/obj/item/food/monkeycube/cube = holder.my_atom
-	if(istype(cube))
-		cube.Expand()
-	else
-		var/location = get_turf(holder.my_atom)
-		new /mob/living/carbon/monkey(location, TRUE)
+
+	var/mob/living/carbon/M = holder.my_atom
+	if(istype(M, /mob/living/carbon))
+		if(istype(M, /mob/living/carbon/monkey))
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/recipes/others.dm:445]: there are helpers for these iirc

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
include discord talk

File: code/modules/reagents/chemistry/recipes/others.dm (hunk 1 above)
Comment (spessbro) [reply code/modules/reagents/chemistry/recipes/others.dm]: yes there are :)

**Pull request merged**